runtime/onert/backend/train/KernelGenerator.cc

```diff
@@ -174,34 +85,10 @@ KernelGenerator::KernelGenerator(const ir::train::TrainableGraph &tgraph,
   // DO NOTHING
 }
 
-std::unique_ptr<exec::train::TrainableFnSequence> KernelGenerator::generate(ir::OperationIndex ind)
-{
-  // TODO Generate TrainableFnSequence that can go backward as well
-  auto ret = std::make_unique<exec::train::TrainableFnSequence>();
-
-  const auto &op = _tgraph.operations().at(ind);
-  // op.accept(*this);
-  // ret->append(releaseFunction());
-
-  for (auto ind : (op.getInputs() | ir::Remove::UNDEFINED) + op.getOutputs())
-  {
-    auto portable_tensor = _tensor_reg->getPortableTensor(ind);
-    if (portable_tensor)
-    {
-      assert(portable_tensor->layout() == ir::Layout::NHWC);
-    }
-    auto tensor = _tensor_reg->getNativeTensor(ind);
-    if (tensor)
-    {
-      tensor->increase_ref();
-    }
-  }
-  return ret;
-}
-
 void KernelGenerator::visit(const ir::train::operation::Loss &)
 {
   // TODO Generate kernel
+  UNUSED_RELEASE(convertPoolType);
```

Comment:
What does this code mean?

Reply:
It was a temporary way to keep `convertPoolType`.